src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -1817,7 +1817,7 @@ await PreviewBuildIntegratedProjectActionsAsync(buildIntegratedProject, actions,
                 PackageSource source = enabledSource.PackageSource;
                 if (source.IsHttp && !source.IsHttps && !source.AllowInsecureConnections)
                 {
-                    nuGetProjectContext.Log(MessageLevel.Warning, Strings.Warning_HttpServerUsage, "install", source.Source);
```

Comment:
This is technically install, not restore.

Reply:
Yes. We pass in the argument "install" to make sure the error is correct.

Reply:
I was referring to the fact that the title says restore.

Reply:
Renamed the title but if the changes are too many we can break it into two PRs as well